lib/index.js

```diff
@@ -43,7 +43,10 @@ function asString(format, date) {
     date = module.exports.now();
   }
 
-  var displayUTC = format.indexOf("O") > -1;
+  // Issue # 14 - Per ISO8601 standard, the time string should be local time
+  // with timezone info.  
+  // See https://en.wikipedia.org/wiki/ISO_8601 section "Time offsets from UTC"
+  var displayUTC = false;
```

Comment:
I think it would be better to get rid of displayUTC altogether - it won't get used any more.